app/assets/javascripts/common/reports.js

```diff
@@ -1312,55 +1353,61 @@ function baseLineGraphConfig() {
           pointStyle: "circle",
           hoverRadius: 5,
         },
+        line: {
+          tension: 0.4,
+          fill: true,
+        }
       },
-      legend: {
-        display: false,
-      },
-      hover: {
+      interaction: {
         mode: "index",
         intersect: false,
       },
+      plugins: {
+        legend: {
+          display: false,
+        },
+      },
       scales: {
-        xAxes: [
-          {
-            stacked: false,
-            display: true,
-            gridLines: {
-              display: false,
-              drawBorder: true,
-            },
-            ticks: {
-              autoSkip: false,
-              fontColor: colors.darkGrey,
-              fontSize: 12,
-              fontFamily: "Roboto",
-              padding: 8,
-              min: 0,
-              beginAtZero: true,
+        x: {
+          stacked: false,
+          display: true,
+          grid: {
+            display: false,
+            drawBorder: true,
+          },
+          ticks: {
+            autoSkip: false,
+            color: colors.darkGrey,
+            font: {
+              size: 12, // default is 12
+              family: "Roboto",
             },
+            padding: 6,
           },
-        ],
-        yAxes: [
-          {
-            stacked: false,
+          beginAtZero: true,
+          min: 0,
+        },
+        y: {
+          stacked: false,
+          display: true,
+          grid: {
             display: true,
-            gridLines: {
-              display: true,
-              drawBorder: false,
-            },
-            ticks: {
-              autoSkip: false,
-              fontColor: colors.darkGrey,
-              fontSize: 10,
-              fontFamily: "Roboto",
-              padding: 8,
-              min: 0,
-              beginAtZero: true,
-              stepSize: 25,
-              max: 100,
+            drawBorder: false,
+          },
+          ticks: {
+            autoSkip: false,
+            color: colors.darkGrey,
+            font: {
+              size: 10,
+              family: "Roboto",
             },
+            padding: 8,
+            stepSize: 25, // unitStepSize??
```

Comment:
Flagging another potential cleanup - or are we addressing this later?

Reply:
This currently is not affecting the graph but is flagged for review - in the upcoming PR